src/message.js

```diff
@@ -0,0 +1,20 @@
+export class Message {
+  constructor (basicMessage, userStore, roomStore) {
+    this.id = basicMessage.id
+    this.senderId = basicMessage.senderId
+    this.roomId = basicMessage.roomId
+    this.text = basicMessage.text
+    this.createdAt = basicMessage.createdAt
+    this.updatedAt = basicMessage.updatedAt
+    this.userStore = userStore
```

Comment:
Yeah the reasoning was that I wanted to keep all the mutable state in the three "stores", and then everything else is always kept up to date for free. I don't _think_ it should be too much of a performance hit, since `getSync` is `O(1)`, but something to try if things get slow though for sure :)